content/millennium/r4/financial/charge-item.md

```diff
@@ -83,13 +83,18 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?  | Type          | Description
------------------------|------------|---------------|-------------------------------------------------------
- `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+ Name                  | Required?       | Type          | Description
+-----------------------|-----------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id     | [`token`]     | Encounter associated with event.
+ `-status`             | see notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
```

Comment:
nit: `see notes` should be capitalized. i.e. `See notes`

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/665/commits/52ec1eb5766e51bfbe059e26fbe147fab674940c#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR90